tillerinobot/src/main/java/tillerino/tillerinobot/osutrack/Downloader.java

```diff
@@ -0,0 +1,34 @@
+package tillerino.tillerinobot.osutrack;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.IOException;
+import java.net.URL;
+
+public class Downloader {
```

Comment:
Can we change this name? I know, let him who is without sin and all, but this is too much.